arangod/Cluster/Maintenance.cpp

```diff
@@ -697,13 +732,8 @@ static VPackBuilder assembleLocalCollectionInfo(
         // in Plan, but in local.
         if (planServers.isArray()) {
           auto current = *(collection->followers()->get());
```

Comment:
`shard_ptr` is only temporary, hence dropped asap. better do
`auto const& current = collection->followers()->get();`
and
`for (auto const& server : current.get())`